pkg/sdn/plugin/controller.go

```diff
@@ -458,7 +462,10 @@ func (plugin *OsdnNode) AddServiceRules(service *kapi.Service, netID uint32) {
 
 	otx := plugin.ovs.NewTransaction()
 	for _, port := range service.Spec.Ports {
+		// Add one flow matching port #s for unfragmented packets and the first fragment
 		otx.AddFlow(generateAddServiceRule(netID, service.Spec.ClusterIP, port.Protocol, int(port.Port)))
+		// Add another flow matching port 0 to allow subsequent fragments through
+		otx.AddFlow(generateAddServiceRule(netID, service.Spec.ClusterIP, port.Protocol, int(0)))
```

Comment:
yeah ive done that locally but its not as simple as just adding ip_frag=later because:

```
 Error executing ovs-ofctl: OFPT_ERROR (OF1.3) (xid=0x2): OFPBMC_BAD_PREREQ
```

still need to check that out